src/features/maintenance/components/maintenance-detail-sheet.tsx

```
@@ -0,0 +1,363 @@
+
+          {/* Actions */}
+          <div className='space-y-4 pt-6 border-t'>
+            {onAssign && !request.assignedTo && (
+              <Button onClick={() => onAssign(request._id)} className='w-full'>
+                Assign Technician
+              </Button>
+            )}
+
+            {onStatusChange && (
+              <div className='space-y-3'>
+                <label className='text-sm text-foreground'>Change Status</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)